models/rollup_economics/ethereum/rollup_economics_ethereum_l1_fees.sql

```diff
@@ -0,0 +1,136 @@
+      evt_block_time AS block_time,
+      evt_block_number AS block_number
+      FROM {{ source('arbitrum_ethereum', 'SequencerInbox_evt_SequencerBatchDeliveredFromOrigin') }} o
+      WHERE evt_block_time >= timestamp '2022-01-01'
+      {% if is_incremental() %}
+          and evt_block_time >= date_trunc('day', now() - interval '7' day)
+      {% endif %}
+
+      UNION ALL
+
+      SELECT
+      call_tx_hash AS tx_hash,
+      call_block_time AS block_time,
+      call_block_number AS block_number
+      FROM {{ source('arbitrum_ethereum','SequencerInbox_call_addSequencerL2BatchFromOrigin') }} o
+      WHERE call_success = true
+      AND call_tx_hash NOT IN
+      (SELECT evt_tx_hash FROM {{ source('arbitrum_ethereum', 'SequencerInbox_evt_SequencerBatchDeliveredFromOrigin') }} o
+      WHERE evt_block_time >= timestamp '2022-01-01'
+      )
+      {% if is_incremental() %}
+          and call_block_time >= date_trunc('day', now() - interval '7' day)
+      {% endif %}
+
+      UNION ALL
+
+      SELECT
+      call_tx_hash AS tx_hash,
+      call_block_time AS block_time,
+      call_block_number AS block_number
+      FROM {{ source('arbitrum_ethereum','SequencerInbox_call_addSequencerL2Batch') }} o
+      WHERE call_success = true
+      AND call_tx_hash NOT IN
+      (SELECT evt_tx_hash FROM {{ source('arbitrum_ethereum', 'SequencerInbox_evt_SequencerBatchDeliveredFromOrigin') }} o
+      WHERE evt_block_time >= timestamp '2022-01-01'
+      )
+      {% if is_incremental() %}
+          and call_block_time >= date_trunc('day', now() - interval '7' day)
+      {% endif %}
+
+      UNION ALL
+
+      SELECT
+      call_tx_hash AS tx_hash,
+      call_block_time AS block_time,
+      call_block_number AS block_number
+      FROM {{ source('arbitrum_ethereum','SequencerInbox_call_addSequencerL2BatchFromOriginWithGasRefunder') }} o
+      WHERE call_success = true
+      AND call_tx_hash NOT IN
+      (SELECT evt_tx_hash FROM {{ source('arbitrum_ethereum', 'SequencerInbox_evt_SequencerBatchDeliveredFromOrigin') }} o
+      WHERE evt_block_time >= timestamp '2022-01-01'
+      )
+      {% if is_incremental() %}
+          and call_block_time >= date_trunc('day', now() - interval '7' day)
+      {% endif %}
+    )b
+    INNER JOIN {{ source('ethereum','transactions') }} t
+    ON b.tx_hash = t.hash
+    WHERE b.block_number = t.block_number
+    AND b.block_time = t.block_time
+    AND success = true
+    AND t.block_time >= timestamp '2022-01-01'
+    {% if is_incremental() %}
+        and block_time >= date_trunc('day', now() - interval '7' day)
+    {% endif %}
+
+    UNION ALL
+    SELECT
+      op.name AS name,
+      t.block_time,
+      t.block_number,
+      t.hash,
+      (cast(gas_used as double) * (cast(gas_price as double) / 1e18)) AS gas_spent,
+      length(t.data) AS data_length
+    FROM
+      {{ source('ethereum','transactions') }} AS t
+      INNER JOIN {{ source('dune_upload','op_stack_chain_metadata') }} op ON (
```

Comment:
nice -- first `dune_upload` i've seen!

Reply:
This comes from tracking in OP Labs Airtable (Devrel manages) -> Dune Upload
https://github.com/ethereum-optimism/op-analytics/blob/main/op_chains_tracking/read_op_chain_metadata_airtable.ipynb